### PR TITLE
remove inner snippets

### PR DESCRIPTION
**Pull request by mairaw**
Related to #1255

[Hide whitespace changes](https://github.com/dotnet/docs/pull/6278/files?utf8=%E2%9C%93&diff=split&w=1) for easier diff

### CODE REVIEW

File: docs/csharp/language-reference/keywords/goto.md [hunk 1]
```diff
@@ -9,28 +9,33 @@ helpviewer_keywords:
...
+## C# language specification
+
+[!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]
+
+## See also
+
+[C# Reference](../index.md)  
+[C# Programming Guide](../../programming-guide/index.md)  
+[C# Keywords](index.md)  
+[goto Statement](/cpp/cpp/goto-statement-cpp)
```
Comment (JRAlexander) [inline docs/csharp/language-reference/keywords/goto.md]: Is this right? a cpp link instead of C#?

**Review by JRAlexander — APPROVED** [review]
LGTM. One nit.

File: docs/csharp/language-reference/keywords/goto.md (hunk 1 above)
Comment (mairaw) [reply docs/csharp/language-reference/keywords/goto.md]: I didn't change this. But given you're already on the goto C# page, it's probably right. Might help to add a suffix there or just remove the link. @BillWagner any preference?

**Comment by mairaw** [comment]
I've decided to keep the link for now and just add the C++ identifier as in previous versions of the topic. Bill, feel free to open a new PR if you prefer to remove that.

**Pull request merged**